WoloxAndroidBootstrp/app/src/main/java/ar/com/wolox/android/example/ui/viewpager/random/RandomNewsRecyclerView.kt

```diff
@@ -0,0 +1,80 @@
+package ar.com.wolox.android.example.ui.viewpager.random
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.model.News
+import ar.com.wolox.android.example.utils.CustomTimeFormat
+import com.bumptech.glide.Glide
+import org.ocpsoft.prettytime.PrettyTime
+import org.ocpsoft.prettytime.units.Minute
+import java.util.ArrayList
+import java.util.Date
+
+class RandomNewsRecyclerView() : RecyclerView.Adapter<RandomNewsRecyclerView.ViewHolder>() {
+
+    private var dataSet = ArrayList<News>()
+
+    private lateinit var mListener: onItemClickListener
```

Comment:
You could also receive this field from the constructor.
`class RandomNewsRecyclerView(listener: onItemClickListener)
`and pass it from the Fragment ;)